spyder_terminal/widgets/main_widget.py

```diff
@@ -9,9 +9,11 @@
 
 # Standard imports
 import os
-import sys
-import requests
 import os.path as osp
+import qstylizer
```

Comment:
`qstylizer` is a third-party import, so please move it to the next section of imports.